traitcuration/traits/datasources/zooma.py

```diff
@@ -0,0 +1,64 @@
+import requests
+
+from ..models import Trait, MappingSuggestion, OntologyTerm, User
+from .ols import make_ols_query, get_ontology_id
+
+
+BASE_URL = "http://www.ebi.ac.uk/spot/zooma/v2/api"
+
+
+def get_zooma_suggestions():
+    """
+    Retrieves zooma mapping suggestions for all traits and creates terms and suggestions in the app's database for each
+    of them.
+    """
+    traits = Trait.objects.all()
+    for trait in traits:
+        results_list = get_suggestions_for_trait(trait)
+        for result in results_list:
+            term = create_local_term(result)
+            if term is not None:
```

Comment:
I think there should be a comment here explaining in which situations a term can be None, and what this implies